languages/ruby/exercises/concept/enumeration/.meta/design.md

```diff
@@ -0,0 +1,19 @@
+## Learning objectives
+
+- Learn what enumeration is
+- Learn how to use enumerable methods
+- Understand how to use a block
+
+## Out of scope
+
+- The yield keyword
+
+## Concepts
+
+- `enumeration`: Know how to use basic enumerable methods (e.g. each, map, find, group_by, sort)
```

Comment:
Each is not, `map`, `find` `group_by`, `sort`, these others are all `Enumerable`.  See comment about "white lies" to see if we can avoid it.  And if we can remove `each` from repeatedly being referenced by us as an "enumerable method".  Surely we should not lie to ourselves though.

Reply:
The methods are not enumerable, but the objects that we use are, perhaps this is the "eh" area.

```suggestion
- `enumeration`: Know how to iterate enumerable objects. (e.g. methods such as `each`, `map`, `find`, `group_by`, `sort`)
```
Because `each` is required to be provided in order to allow an object to become enumerable, once provided, we can then `include Enumerable`, and whatever object includes that becomes an `Enumerable` type, while the `each` method itself is not from that module.  But we can iterate enumerable objects with that method... I think it avoids the "lie" that happens pretty commonly, enough that many people believe it much longer than they should.